react/features/settings/functions.js

```diff
@@ -60,3 +62,50 @@ export function shouldShowOnlyDeviceSelection() {
     return interfaceConfig.SETTINGS_SECTIONS.length === 1
         && isSettingEnabled('devices');
 }
+
+/**
+ * Returns the properties for the "More" tab from settings dialog from Redux
+ * state.
+ *
+ * @param {(Function|Object)} stateful -The (whole) redux state, or redux's
+ * {@code getState} function to be used to retrieve the state.
+ * @returns {Object} - The properties for the "More" tab from settings dialog.
+ */
+export function getMoreTabProps(stateful: Object | Function) {
+    const state = toState(stateful);
+    const language = i18next.language || DEFAULT_LANGUAGE;
+    const conference = state['features/base/conference'];
+    const configuredTabs = interfaceConfig.SETTINGS_SECTIONS || [];
+
+    return {
+        currentLanguage: language,
+        followMeEnabled: Boolean(conference.followMeEnabled),
+        languages: LANGUAGES,
+        showLanguageSettings: configuredTabs.includes('language'),
+        showModeratorSettings: configuredTabs.includes('moderator'),
```

Comment:
This also should check for if the user is a moderator. That's why the tests are failing, I think.

Reply:
Fixed!